spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/plugin/SpringBootPluginTests.java

```diff
@@ -35,7 +35,7 @@
  * @author Andy Wilkinson
  */
 @ClassPathExclusions("kotlin-daemon-client-*.jar")
-public class SpringBootPluginTests {
```

Comment:
I think we'd need a public specifier here.

Reply:
@akashchandwani It's JUnit Jupiter-based, so I thought package-private is sufficient. Am I missing something?

Reply:
Oh Sorry, my bad...
Just checked other files in this package, they don't have `public` specifier, so I guess we'd be fine :)